From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Thu, 6 Feb 2014 13:35:16 +0000
Subject: man macros

Avoid lintian error

  qhull-bin: manpage-has-errors-from-man

to whit

  usr/share/man/man1/qhull.1.gz 750: warning: macro `QJn'' not defined
  usr/share/man/man1/qhull.1.gz 771: warning: macro `Qt'' not defined

and consequent lost text.  These result from a line starting with the
single quote character; this patch backslash escapes them.
---
 html/qhull.man | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/html/qhull.man b/html/qhull.man
index 4ede521..780d706 100644
--- a/html/qhull.man
+++ b/html/qhull.man
@@ -747,7 +747,7 @@ of attempts [see user.h].  Option 'QJ'
 selects a default value for n.  The output will be simplicial.  For
 Delaunay triangulations, 'QJn' sets 'Qbb' to scale the last coordinate
 (not if 'Qbk:n' or 'QBk:n' is set).
-'QJn' is deprecated for Voronoi diagrams.  See also 'Qt'.
+\'QJn' is deprecated for Voronoi diagrams.  See also 'Qt'.
 .TP
 Qm
 Only process points that would otherwise increase max_outside.  Other
@@ -769,7 +769,7 @@ Search all points for the initial simplex.
 .TP
 Qt
 Triangulated output.  Triangulate all non\[hy]simplicial facets.  
-'Qt' is deprecated for Voronoi diagrams.  See also 'Qt'.
+\'Qt' is deprecated for Voronoi diagrams.  See also 'Qt'.
 .TP
 Qv
 Test vertex neighbors for convexity after post\[hy]merging.
